device-connectors/src/testflinger_device_connectors/devices/zapper_iot/__init__.py

```diff
@@ -56,16 +60,44 @@ def _validate_configuration(
         if provision_plan:
 
             try:
-                validate_provision_plan(provision_plan)
-
                 # Make sure the user created at provision time is
                 # the same used during the test phase.
                 provision_plan["config"]["username"] = username
                 provision_plan["config"]["password"] = password
```

Comment:
And if you liked my suggestion above, you should also apply this change here.
```suggestion
                provision_plan["config"]["username"] = provision_data["username"]
                provision_plan["config"]["password"] = provision_data["password"]
```

Reply:
Thanks for double checking others I need to modify.